pyworkflow/plugin.py

```diff
@@ -131,19 +134,17 @@ def getPlugins(cls):
     @classmethod
     def _discoverPlugins(cls):
         # Get the list of plugins registered
-        plugin_modules = []
-        for entry_point in pkg_resources.iter_entry_points('pyworkflow.plugin'):
-            plugin_modules.append(entry_point.name)
+        plugin_modules = importlib_metadata.entry_points(group='pyworkflow.plugin')
```

Comment:
I was testing this this morning. My entry_points method does not have any argument ?